catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/ListApplication.java

```diff
@@ -181,10 +224,11 @@ private boolean attemptToSplitAndStore(
   private void storeAndClose(
       Consumer<String> idConsumer,
       Consumer<String> errorMessageConsumer,
-      StorableResource storableResource) {
+      StorableResource storableResource,
+      Metacard right) {
```

Comment:
Naming the variable `right` doesn't seem like a descriptive name. How about just calling it `metacard`?